tensorboard/build.zig

```diff
@@ -0,0 +1,53 @@
+const std = @import("std");
+
+pub fn build(b: *std.Build) void {
+    const target = b.standardTargetOptions(.{});
+    const optimize = b.standardOptimizeOption(.{});
+
+    const protobuf_dep = b.dependency("protobuf", .{
+        .target = target,
+        .optimize = optimize,
+    });
+
+    const tensorboard = b.addModule("tensorboard", .{
+        .root_source_file = b.path("src/root.zig"),
+        .target = target,
+        .optimize = optimize,
+    });
+    tensorboard.addImport("protobuf", protobuf_dep.module("protobuf"));
```

Comment:
Interesting that I do not need `lib.root_module.addImport("protobuf", protobuf_dep.module("protobuf"));` it seems this module becomes the root module of the static library. Only connection between the two I see is the `root_source_file` it seems Zig is quite clever or I am missing something.

Reply:
Interesting - yeah I can see that it's not being passed in there. Are you using that static library somewhere else? Not sure how that linkage is working (if it is?).

Reply:
Yeah I originally had that line there and it worked both as a package dependency and the tests passed so within the package itself. After deleting it, still worked. Maybe I'll put it back in just bc it makes more sense.